src/assets/custom/css/_sass/_announcement-slice.scss

```diff
@@ -1,42 +1,45 @@
 .announcement-slice {
-  background-color: $coral;
+  background-image: url("#{$site-base-url}/assets/custom/img/services/software-modernisation/bg-announcement-slice-softmod.jpg");
   color: white;
-  line-height: 34px;
-  padding-bottom: 10px;
-  padding-top: 10px;
+  padding-bottom: 15px;
+  padding-top: 15px;
   position: absolute;
   width: 100%;
-  z-index: 5;
+  z-index: 9999;
+  background-size: cover;
+  line-height: 1.5em;
+  text-align: center;
+
+  @media (min-width: 580px) {
```

Comment:
It's a shame that we are introducing a new break point here? @Dan-Bird can show you how we you could redo this using `small` and `medium-large-and-extra-large` if you would be happy to use `768px` instead.

If 768px isn't right we should consider changing the medium breakpoint. It would be worth trying to avoid introducing new adhoc media queries

Reply:
Resolved here: https://github.com/codurance/site/pull/1924/commits/6c05d36e4190cd15f9dbfb11a569716e1bce7914